app.py

```diff
@@ -58,7 +58,7 @@ def create_app(env):
     create_routes(app)
 
     #allow cross-origin (CORS)
-    CORS(app)
+    CORS(app, origins=app.config['CORS_ORIGINS'])
```

Comment:
If we change this to:
```suggestion
    CORS(app, origins=[app.config['CORS_ORIGINS'])]
```
Do we still need to have the regex on the config var?

Reply:
The regex is to allow for multiple separators in the case where multiple origins are set in the environmental variable. So "origin1,origin2" and "origin1;origin2" or "origin1 origin2" are all permitted. But if we move the the configuration into Development, Testing, and Production classes then the regex is not necessary.